### ISSUE TITLE
Create a Release Plan with features and dates

### ISSUE THREAD

**Issue by francescmm** [francescmm/GitQlient#89]
There is now written plan for the releases, when they are going to be and what features are planned for them. Since GitQlient is getting more used, and to improve visibility I'd be nice to have it somewhere in the repo. Perhaps a wiki page.

This should be done before finishing v1.1.0.

This is caused by #84.

**Comment by francescmm** [comment]
Added a new page in the [Wiki](https://github.com/francescmm/GitQlient/wiki/Planning)
